mypy/options.py

```diff
@@ -132,6 +133,9 @@ def __init__(self) -> None:
         # Files in which to ignore all non-fatal errors
         self.ignore_errors = False
 
+        # Ignore errors with these error codes in a given file.
```

Comment:
Nit: "in a given files." seems redunant. Also omit period at the end for consistency?